src/libraries/System.Runtime.Numerics/tests/BigIntegerTests.GenericMath.cs

```diff
@@ -366,6 +366,9 @@ public static void TrailingZeroCountTest()
 
             Assert.Equal((BigInteger)63, BinaryIntegerHelper<BigInteger>.TrailingZeroCount(Int64MaxValuePlusOne));
             Assert.Equal((BigInteger)0, BinaryIntegerHelper<BigInteger>.TrailingZeroCount(UInt64MaxValue));
+
+            Assert.Equal((BigInteger)1000, BinaryIntegerHelper<BigInteger>.TrailingZeroCount(BigInteger.Pow(2, 1000)));
+            Assert.Equal((BigInteger)1000, BinaryIntegerHelper<BigInteger>.TrailingZeroCount(BigInteger.Pow(-2, 1000)));
```

Comment:
`BigInteger.Pow(-2, 1000)` is the same as `BigInteger.Pow(2, 1000)`, because an even power of a negative base is positive.
```suggestion
            Assert.Equal((BigInteger)1000, BinaryIntegerHelper<BigInteger>.TrailingZeroCount(-BigInteger.Pow(2, 1000)));
```

Reply:
Whoops, thank you for the catch, you are completely correct.